gradle/dependencies.gradle

```diff
@@ -234,6 +227,7 @@ libs += [
   kafkaStreams_34: "org.apache.kafka:kafka-streams:$versions.kafka_34",
   kafkaStreams_35: "org.apache.kafka:kafka-streams:$versions.kafka_35",
   kafkaStreams_36: "org.apache.kafka:kafka-streams:$versions.kafka_36",
+  kerbSimpleKDC: "org.apache.kerby:kerb-simplekdc:2.0.3",
```

Comment:
when we are introducing a new dependency, we need to add it to https://github.com/apache/kafka/blob/cfc8257479c37e02344689405954b01fcc8f24f7/LICENSE-binary#L205

Similarly, we need to modify the version of apacheda and apacheds specified there.

Also, please move the version to `versions` dictionary above.